### PR TITLE
fix(RemotePlayback): use APIRef + add GroupData for overview page

### PR DESCRIPTION
**Pull request by caugner**
### Description

Uses the APIRef sidebar for all "RemotePlayback" pages, and adds a GroupData item for the DefaultAPISidebar on the overview page.

### Motivation

Makes sure the following pages have a sidebar:

-   /en-US/docs/Web/API/Remote_Playback_API
-   /en-US/docs/Web/API/RemotePlayback/cancelWatchAvailability
-   /en-US/docs/Web/API/RemotePlayback
-   /en-US/docs/Web/API/RemotePlayback/prompt
-   /en-US/docs/Web/API/RemotePlayback/state
-   /en-US/docs/Web/API/RemotePlayback/watchAvailability

### Related issues and pull requests

Part of https://github.com/mdn/content/issues/22441.

### CODE REVIEW

File: files/jsondata/GroupData.json [hunk 1]
```diff
@@ -1127,6 +1127,21 @@
...
+      "methods": [
+        "RemotePlayback.watchAvailability()",
+        "RemotePlayback.cancelWatchAvailability()",
+        "RemotePlayback.prompt()"
+      ],
+      "properties": ["RemotePlayback.state"],
+      "events": [
+        "RemotePlayback: connecting",
+        "RemotePlayback: connect",
+        "RemotePlayback: disconnect"
```
Comment (wbamberg) [inline files/jsondata/GroupData.json:1142]: These should not list members of interfaces that appear in "interfaces", it's for members that are part of other interfaces, which in this case seems to be the extensions to `HTMLMediaElement`: https://w3c.github.io/remote-playback/#extensions-to-htmlmediaelement.

File: files/en-us/web/api/remoteplayback/cancelwatchavailability/index.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ page-type: web-api-instance-method
 browser-compat: api.RemotePlayback.cancelWatchAvailability
 ---
 
-{{DefaultAPISidebar("Remote Playback API")}}
+{{APIRef}}
```
Comment (wbamberg) [inline files/en-us/web/api/remoteplayback/cancelwatchavailability/index.md:9]:
```suggestion
{{APIRef("Remote Playback API")}}
```

(same goes for all the others in this PR)

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks!

Apart from the inline comments, I guess we should add `{{DefaultAPISidebar("Remote Playback API")}}` to https://developer.mozilla.org/en-US/docs/Web/API/Remote_Playback_API ?

**Comment by bsmth** [comment]
Removing myself from review while this is in a draft state, feel free to add me again if you want me to take a look when it's ready :)

**Comment by caugner** [comment]
Closing, as I don't have capacity to work on this in the near future. But I appreciate Will's review. 🙌

**Pull request closed without merging**